src/Goerli-DssSpellCollateral.sol

```diff
@@ -32,79 +35,61 @@ contract DssSpellCollateralAction {
     //
 
     // --- Math ---
-    // uint256 constant THOUSAND   = 10 ** 3;
-    // uint256 constant MILLION    = 10 ** 6;
-    // uint256 constant BILLION    = 10 ** 9;
+    uint256 constant MILLION  = 10 ** 6;
+    uint256 constant THOUSAND = 10 ** 3;
+    uint256 constant ONE_FIVE_PCT_RATE        = 1000000000472114805215157978;
 
     // --- DEPLOYED COLLATERAL ADDRESSES ---
-    // address constant XXX                  = 0x0000000000000000000000000000000000000000;
-    // address constant PIP_XXX              = 0x0000000000000000000000000000000000000000;
-    // address constant MCD_JOIN_XXX_A       = 0x0000000000000000000000000000000000000000;
-    // address constant MCD_CLIP_XXX_A       = 0x0000000000000000000000000000000000000000;
-    // address constant MCD_CLIP_CALC_XXX_A  = 0x0000000000000000000000000000000000000000;
-
-    // --- Offboarding: Current Liquidation Ratio ---
-    // uint256 constant CURRENT_XXX_A_MAT              =  XYZ * RAY / 100;
-
-    // --- Offboarding: Target Liquidation Ratio ---
-    // uint256 constant TARGET_XXX_A_MAT               =  XYZ * RAY / 100;
-
+    address constant RETH                     = 0x62BC478FFC429161115A6E4090f819CE5C50A5d9;
```

Comment:
Same thing here, existing pattern is to use internal as visibility for constants. Just a nit :)